.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
```

Comment:
Not important for this PR, but I wonder if this will be a pain to update as we update Python versions.

Reply:
I agree. Though, this replaces the previous implementation which used an exclude arch for each type we don't intend to include --- I think neither are perfect, but I think this can be something to revisit in future.

Reply:
I would say inclusion is better than exclusion :+1:

Better to manually decide one day to support Python 3.10, rather than discovering that one day PR CI is failing because 3.10 builds got turned on automatically!